DevelopGitHub/GitHubExtension/GitHubExtension.DB/Activities.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Activities]
+(
+	[Id] INT NOT NULL IDENTITY(1,1), 
```

Comment:
In visual studio code in sql files is ok, but when push it to github, it's dancing, just don't know why =)